beets/util/__init__.py

```diff
@@ -216,8 +216,7 @@ def mkdirall(path):
             try:
                 os.mkdir(syspath(ancestor))
             except (OSError, IOError) as exc:
-                raise FilesystemError(exc, 'create', (ancestor,),
-                                      traceback.format_exc())
```

Comment:
this turned out to be waaay simpler than I thought.. seriously.  You're gonna laugh probably!.  Take a look here: https://github.com/beetbox/beets/blob/18b57ea02260039e76493b2baa4b1a3cefb17f15/beets/util/__init__.py#L446  and you'll see that there's no traceback for us to format. Python 2.x (and seemingly 3.5+)  are just letting us format non-existent tracebacks!